www/js/app.js

```diff
@@ -1322,7 +1321,6 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
             Array.prototype.slice.call(iframe.querySelectorAll('a, area')).forEach(function (anchor) {
                 // Attempts to access any properties of 'this' with malformed URLs causes app crash in Edge/UWP [kiwix-js #430]
                 try {
-                    var testHref = anchor.href;
```

Comment:
If you take a look at the issue mentioned in the comment, you will see why this line is necessary and must not be removed!